cadasta/api/test/test_login.py

```diff
@@ -4,32 +4,31 @@
 
 import unittest
 
+import os
 import qgis
 import logging
 from cadasta.api.login import Login
-from cadasta.test.utilities import get_qgis_app
 from qgis.PyQt.QtCore import QCoreApplication
 
-
-QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
-
-LOGGER = logging.getLogger('CadastaQGISPlugin')
+if not os.environ.get('ON_TRAVIS', False):
+    from cadasta.test.utilities import get_qgis_app
+    QGIS_APP = get_qgis_app()
```

Comment:
Any reason why you don't want the QGIS app when you are not in travis ? How do you do in pycharm ?